acceptance/panic/output.txt

```diff
@@ -0,0 +1,13 @@
+
+>>> [CLI] panic
+The Databricks CLI unexpectedly panicked.
```

Comment:
Panic is a golang-specific term that users are not required to know, perhaps "The Databricks CLI had fatal error"?

Reply:
Either way works. The next line does direct the user to open an issue on Databricks so they don't need to know what panicking means.

Reply:
+1 for avoiding implementation terminology

Reply:
fixed.